cloud_functions/token_refresh/streams/strava.py

```
@@ -0,0 +1,79 @@
+            "lastTokenRefresh": update_last_refresh(),
+        }
+
+        strava_update_user_tokens(db=db, uid=uid, new_tokens=new_tokens)
+        logger.info("Token has been refreshed")
+    else:
+        logger.info("Token is still valid")
+
+
+@beartype
+def strava_update_user_tokens(
+    db: firestore.Client, uid: str, new_tokens: Dict[str, Any]
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)